iothub/service/src/Common/Security/CryptoKeyGenerator.cs

```diff
@@ -73,6 +83,10 @@ public static string GenerateKeyInHex(int keySize)
             return BitConverter.ToString(keyBytes).Replace("-", "");
         }
 
+        /// <summary>
+        /// Generate a unique GUID
```

Comment:
It isn't just a unique GUID, because otherwise it would be `Guid.NewGuid()`. What kind of GUID does it create?

Reply:
This method generates a guid using bytes generated from the RNGCryptoServiceProvider class. Will update to reflect that.